File: k-kbk/2주차/2816.py
n = int(input())

channels = [input() for _ in range(n)]

result = ""

cursor = 0
kbs1 = channels.index("KBS1")
result += "1" * kbs1
result += "4" * kbs1

channels.remove("KBS1")
channels.insert(0, "KBS1")

kbs2 = channels.index("KBS2")
result += "1" * kbs2
result += "4" * (kbs2 - 1)

print(result)

File: keemsebin/2주차/2816.js
const readline = require("readline");
const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout,
});

let input = [];

rl.on("line", function (line) {
  input.push(line);
}).on("close", function () {
  let N = parseInt(input[0]);
  let arr = input.slice(1);
  let result = "";

  let posKBS1 = arr.indexOf("KBS1");
  let posKBS2 = arr.indexOf("KBS2");
  posKBS2 = posKBS1 > posKBS2 ? ++posKBS2 : posKBS2;

  for (let i = 0; i < posKBS1; i++) {
    result += "1";
  }
  for (let i = 0; i < posKBS1; i++) {
    result += "4";
  }

  for (let i = 0; i < posKBS2; i++) {
    result += "1";
  }
  for (let i = 0; i < posKBS2 - 1; i++) {
    result += "4";
  }

  console.log(result);
});
